### PR TITLE
Preprod: Theme definition work + story file updates

### PR DESCRIPTION
**Pull request by paul-sheridan**
Change log:
- Defined **piloted creature-shaped vehicle** theme
- Defined **are humans more than machines** theme
- Revised **speculative robot** theme entry
- Updated story files to reflect that **mecha robot** is replaced by **piloted creature-shaped vehicle**

### CODE REVIEW

File: notes/themes/proposed.th.txt [hunk 1]
```diff
@@ -506,5 +506,57 @@ human mental condition
...
+
+:: References
+https://en.wikipedia.org/wiki/Robots_in_literature
+
+
+piloted creature-shaped vehicle
+===============================
+
+:: Description
+There is a belimbed (or betentacled etc.) mobile automaton that is piloted by
```
Comment (odinlake) [inline notes/themes/proposed.th.txt:537]: I don't think "automaton" is the right word here.

Comment (odinlake) [reply notes/themes/proposed.th.txt:537]: Personally I think a tentacle is a kind of limb, but I guess we should consult a marine biologist and a linguist about that... 🙃

**Review by odinlake — APPROVED** [review]
I think the word "automaton" should be replaced, but do as you please.

File: notes/themes/proposed.th.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/themes/proposed.th.txt:537]: I guess something like a golem might be considered to be an automaton. What would you use in place of automaton? mechanical contraption?

**Comment by odinlake** [comment]
“Machine”?

**Comment by paul-sheridan** [comment]
Replaced "automaton" by "machine".Squash and merge imminent.

**Pull request merged**